Wikipedia/Code/WMFApiJsonResponseSerializer.m

```diff
@@ -27,6 +27,14 @@ - (id)responseObjectForResponse:(NSURLResponse *)response
         }
         return nil;
     }
+
+#if DEBUG
+    NSDictionary *warnings = json[@"warnings"];
+    if (warnings) {
+        NSLog(@"\n\n\nWarnings for request:\n%@\n%@", response.URL.absoluteString, warnings);
```

Comment:
I suppose we could also assert that there are no warnings... would that be preferred?

Reply:
I think I'd rather just log for now - I think there were a few cases of expected warnings in places search. Also use `DDLogWarn` for consistency (even though it's wrapped in #if DEBUG)

Reply:
Ah will do.

Reply:
Fixed!